src/backend/vector/mod.rs

```diff
@@ -11,29 +11,29 @@
 
 #![doc = include_str!("../../../docs/parallel-formulas.md")]
 
-#[cfg(not(any(target_feature = "avx2", target_feature = "avx512ifma", rustdoc)))]
+#[cfg(not(any(target_feature = "avx2", target_feature = "avx512ifma", docsrs)))]
 compile_error!("simd_backend selected without target_feature=+avx2 or +avx512ifma");
 
 #[cfg(any(
     all(target_feature = "avx2", not(target_feature = "avx512ifma")),
-    rustdoc
+    docsrs
 ))]
-#[doc(cfg(all(target_feature = "avx2", not(target_feature = "avx512ifma"))))]
```

Comment:
doc-cfg is annoying unstable nightly feature and we use cfg's for this same thing already.